docs/docusaurus.config.js

```diff
@@ -89,7 +89,7 @@ module.exports = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           editUrl:
-            'https://github.com/anitab-org/bridge-in-tech-backend/docs',
+            'https://github.com/anitab-org/bridge-in-tech-backend/edit/develop/docs/',
```

Comment:
```suggestion
            'https://github.com/anitab-org/bridge-in-tech-backend/edit/develop/docs',
```
trailing slash is not required. Sorry if I have added previously.

Reply:
@codesankalp Sure, done with changes.